--------- beginning of /dev/log/main
W/dalvikvm( 2097): threadid=1: thread exiting with uncaught exception (group=0x40019560)
--------- beginning of /dev/log/system
E/AndroidRuntime( 2097): FATAL EXCEPTION: main
E/AndroidRuntime( 2097): java.lang.SecurityException: Neither user 10015 nor current process has android.permission.REBOOT.
E/AndroidRuntime( 2097): 	at android.os.Parcel.readException(Parcel.java:1322)
E/AndroidRuntime( 2097): 	at android.os.Parcel.readException(Parcel.java:1276)
E/AndroidRuntime( 2097): 	at android.os.IPowerManager$Stub$Proxy.reboot(IPowerManager.java:560)
E/AndroidRuntime( 2097): 	at android.os.PowerManager.reboot(PowerManager.java:532)
E/AndroidRuntime( 2097): 	at com.crazyj36.apiplaygroundanysdk.MainActivity$14.onClick(MainActivity.java:384)
E/AndroidRuntime( 2097): 	at android.view.View.performClick(View.java:2520)
E/AndroidRuntime( 2097): 	at android.view.View$PerformClick.run(View.java:9157)
E/AndroidRuntime( 2097): 	at android.os.Handler.handleCallback(Handler.java:587)
E/AndroidRuntime( 2097): 	at android.os.Handler.dispatchMessage(Handler.java:92)
E/AndroidRuntime( 2097): 	at android.os.Looper.loop(Looper.java:130)
E/AndroidRuntime( 2097): 	at android.app.ActivityThread.main(ActivityThread.java:3694)
E/AndroidRuntime( 2097): 	at java.lang.reflect.Method.invokeNative(Native Method)
E/AndroidRuntime( 2097): 	at java.lang.reflect.Method.invoke(Method.java:507)
E/AndroidRuntime( 2097): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:895)
E/AndroidRuntime( 2097): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:653)
E/AndroidRuntime( 2097): 	at dalvik.system.NativeStart.main(Native Method)
W/ActivityManager(  283):   Force finishing activity com.crazyj36.apiplaygroundanysdk/.MainActivity
W/ActivityManager(  283): Activity pause timeout for HistoryRecord{40784dc8 com.crazyj36.apiplaygroundanysdk/.MainActivity}
